readme.rst

```diff
@@ -55,6 +55,17 @@ The command:
 the more standard ``make html``. For instance, ``PROJECT=azure make html`` will create the 
 azure related html pages in _build/azure.)
 
+Check compliance with contribution guides for each project 
```

Comment:
Since they are guidelines checking 'compliance' with them seems a little too prescriptive. They could be used for further/specific guidance as required. So maybe instead of adding it under the 'How to work with this documentation' section, let's create a separate section at the end titled 'Per project contribution guidelines' or something like that.

Reply:
done